test/AutoRest.TestServerLowLevel.Tests/LRO_LowLevelTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using NUnit.Framework;
+using LRO_LowLevel;
+using System.Linq;
+
+namespace AutoRest.TestServer.Tests
+{
+    public class LRO_LowLevelTests
+    {
+        [TestCase("DoSomethingSlroAsync")]
+        [TestCase("DoSomethingSlro")]
+        [TestCase("PageableSlroAsync")]
+        [TestCase("PageableSlro")]
+        public void ValidateSLROMethods(string methodName)
+        {
+            var client = typeof(LROServiceClient);
+            var method = client.GetMethod(methodName);
+            Assert.False(method.Name.StartsWith("Start"));
+
+            var waitForCompletionParam = method.GetParameters().Where(P => P.Name == "waitForCompletion").First();
+            Assert.NotNull(waitForCompletionParam);
+            Assert.AreEqual(waitForCompletionParam.ParameterType, typeof(bool));
+            Assert.AreEqual(waitForCompletionParam.IsOptional, true);
+            Assert.AreEqual(waitForCompletionParam.DefaultValue, true);
+        }
+
+        [TestCase("StartDoSomethingLROAsync")]
+        [TestCase("StartDoSomethingLRO")]
+        [TestCase("StartPageableLROAsync")]
+        [TestCase("StartPageableLRO")]
```

Comment:
Here are the unit tests to verify Short LRO, Long LRO, Pageable Short LRO, Pageable Long LRO methods.